festflow/models.py

```diff
@@ -1,7 +1,10 @@
 import hashlib
 
 from django.contrib.auth.models import User
-from django.core.validators import RegexValidator
+from django.contrib.sites.models import Site
```

Comment:
I dont remember but did we include all Sites related settings in the settings.py?